dor/providers/parsers.py

```diff
@@ -59,45 +65,51 @@ def get_data_files(self) -> list[FileMetadata]:
             for elem in self.tree.findall(".//METS:file")
         ]
 
-    def get_md_file_metadatum(self, elem):
+    def get_md_file_metadatum(self, elem: ElementAdapter):
         id_ = elem.get("ID")
         use = elem.get("USE")
-        locref = elem.find("METS:mdRef").get_optional("LOCREF")
-        mdtype = elem.find("METS:mdRef").get_optional("MDTYPE")
-        mimetype = elem.find("METS:mdRef").get_optional("MIMETYPE")
+        md_red_element = elem.find("METS:mdRef")
+        
+        if md_red_element:
```

Comment:
See comment above about `elem.find` raising an exception, so this check `if md_red_element` shouldn't be necessary.